tests/cmdline/repl_emacs_words_move.py

```diff
@@ -0,0 +1,9 @@
+# word movement
```

Comment:
This doesn't cover all cases I think, i.e the cases where you return early because of reaching the end of the line?

Reply:
Yeah, it doesn't, still WIP :)

Reply:
Now it covers, I think